### PR TITLE
feat: use stdout when encrypt and decrypt data

### LINKED ISSUE #554: use stdout instead of `output-filepath` for CLI `encrypt/decrypt-data`
if other outputs don't pollute stdout already

### PR DESCRIPTION
**Pull request by inchori**
Closes #554

### CODE REVIEW

**Comment by inchori** [comment]
I'll re-open this PR when `decrypt-data` test done.

**Review by gyuguen — APPROVED** [review]
LGTM
I left a comment related to typo

File: cmd/panacead/cmd/decrypt_data.go [hunk 1]
```diff
@@ -52,15 +52,14 @@ func DecryptDataCmd(defaultNodeHome string) *cobra.Command {
...
 				return err
 			}
 
-			err = os.WriteFile(args[1], decryptedData, 0644)
-			if err != nil {
-				return err
-			}
+			decryptedDataBase64 := base64.StdEncoding.EncodeToString(decryptedData)
+
+			cmd.Println(decryptedDataBase64)
```
[the remark this answers is not in the document]
Comment (inchori) [reply cmd/panacead/cmd/decrypt_data.go]: Fixed in [0b0451c](https://github.com/medibloc/panacea-core/pull/561/commits/0b0451c28dea734ca2ab392adb490b1eaf953ae1).

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Review by H4NLee — APPROVED** [review]
lgtm

**Review by audtlr24 — APPROVED** [review]
lgtm

**Pull request merged**